portal/migrations/0002_eventsobject.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 5.0.2 on 2024-09-11 15:31
```

Comment:
Seems like you are using old django version. On dev it's 5.1.1 now

Reply:
Thanks for the warning. This was generated before we merged that change.